pyproject.toml

```diff
@@ -21,6 +21,7 @@ dnspython = "^2.2.1"
 pydantic = "^1.9.0"
 envtoml = "^0.1.2"
 requests = "^2.27.0"
+statsmodels = "^0.13.2"
```

Comment:
This is not needed in the "main" dependencies since it runs only inside monetdb.

You can move it to dev-dependencies so it will work locally as well.